pkg/utils/tekton/controller.go

```diff
@@ -161,6 +161,47 @@ func (s *SuiteController) GetPipelineRunLogs(pipelineRunName, namespace string)
 	return podLog, nil
 }
 
+func (s *SuiteController) GetTaskRunLogs(pipelineRunName, taskName, namespace string) (map[string]string, error) {
+	tektonClient := s.PipelineClient().TektonV1().PipelineRuns(namespace)
```

Comment:
there's an issue when using the `TektonV1` client
```suggestion
	tektonClient := s.PipelineClient().TektonV1beta1().PipelineRuns(namespace)
```

Reply:
Thank you, didn't know that. Will fix.